packages/build-tools/bin/rivet.sh

```
@@ -0,0 +1,133 @@
+    if is_foundry_installed; then
+        forge --version 2>/dev/null | head -n1 | grep -o 'v[0-9]\+\.[0-9]\+\.[0-9]\+' | head -n1
+    else
+        echo ""
+    fi
+}
+
+# Function to install foundry
+install_foundry() {
+    echo "🔧 Foundry not found. Installing foundry..."
+    curl -L https://foundry.paradigm.xyz | bash
+    source ~/.bashrc 2>/dev/null || true
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)